File: Announcement.Core/Models/Ratings.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Announcement.Core
{
    public class Ratings 
    {
        [JsonProperty("topUsers")]
        public List<User> TopUsers { get; set; }

        [JsonProperty("topSpammers")]
        public List<Spammer> TopSpammers { get; set; }
    }
}

File: workspace/api-stop-spam/src/main/java/api/Ratings.java
package api;

import api.Spammer;
import api.User;
import java.util.List;

public class Ratings 
{
    private List<User> topUsers;
    
    private List<Spammer> topSpammers;

    public List<User> getTopUsers() 
    {
        return this.topUsers;
    }

    public void setTopUsers(List<User> topUsers) 
    {
        this.topUsers = topUsers;
    }

    public List<Spammer> getTopSpammers() 
    {
        return this.topSpammers;
    }

    public void setTopSpammers(List<Spammer> topSpammers) 
    {
        this.topSpammers = topSpammers;
    }

    public Ratings() 
    {
    	
    }

    public Ratings(List<User> topUsers, List<Spammer> topSpammers) 
    {
        this.topUsers = topUsers;
        
        this.topSpammers = topSpammers;
    }
}